### PR TITLE
BAEL-5321: Change Swagger UI URL prefix

### PR DESCRIPTION
**Pull request by sdhiray7**
Added code to demonstrate implementation of Springdoc and Springfox.

### CODE REVIEW

File: spring-boot-modules/spring-boot-swagger-2/src/main/java/com/baeldung/springdoc/demo/config/SpringDocSwaggerConfig.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+@Configuration
+public class SpringDocSwaggerConfig {
+
+	@Configuration
+	@EnableSwagger2
+	public class SwaggerConfig {
```
Comment (JavierPeris) [inline spring-boot-modules/spring-boot-swagger-2/src/main/java/com/baeldung/springdoc/demo/config/SpringDocSwaggerConfig.java:17]: When declaring an inner class, it's generally recommended to make it static if it doesn't need to access instance-specific data or methods of the outer class. So could you

File: spring-boot-modules/spring-boot-swagger-2/src/main/java/com/baeldung/springdoc/demo/config/SpringFoxSwaggerConfig.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                "Sample API REST",
+                "API REST",
+                "1.0",
+                "Terms of Service",
+                null,
+                "Apache License Version 2.0",
+                "https://www.apache.org/licesen.html", new ArrayList<>()
+        );
+
+        return apiInfo;
```
Comment (JavierPeris) [inline spring-boot-modules/spring-boot-swagger-2/src/main/java/com/baeldung/springdoc/demo/config/SpringFoxSwaggerConfig.java:39]: You can refactor this code to inline variable.

File: spring-boot-modules/spring-boot-swagger-2/src/main/java/com/baeldung/springdoc/demo/model/Topic.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+	}
+	public void setName(String name) {
+		this.name = name;
+	}
+	public String getDescription() {
+		return description;
+	}
+	public void setDescription(String description) {
+		this.description = description;
+	}
```
[the remark this answers is not in the document]
Comment (sdhiray7) [reply spring-boot-modules/spring-boot-swagger-2/src/main/java/com/baeldung/springdoc/demo/model/Topic.java]: done

File: spring-boot-modules/spring-boot-swagger-2/src/main/java/com/baeldung/springdoc/demo/config/SpringFoxSwaggerConfig.java (hunk 2 above)
Comment (sdhiray7) [reply spring-boot-modules/spring-boot-swagger-2/src/main/java/com/baeldung/springdoc/demo/config/SpringFoxSwaggerConfig.java]: done

File: spring-boot-modules/spring-boot-swagger-2/src/main/java/com/baeldung/springdoc/demo/config/SpringDocSwaggerConfig.java (hunk 1 above)
Comment (sdhiray7) [reply spring-boot-modules/spring-boot-swagger-2/src/main/java/com/baeldung/springdoc/demo/config/SpringDocSwaggerConfig.java]: removed

File: spring-boot-modules/spring-boot-swagger-2/src/main/java/com/baeldung/springdoc/demo/controller/TopicsController.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+@RestController
+@RequestMapping(path = "/")
+public class TopicsController {
+	
+	List<Topic> topics = new ArrayList<Topic>() {{
+		add(new Topic(1, "Topic1"));
+		add(new Topic(2, "Topic2"));
+		add(new Topic(3, "Topic3"));
+	}};
```
[the remark this answers is not in the document]
Comment (sdhiray7) [reply spring-boot-modules/spring-boot-swagger-2/src/main/java/com/baeldung/springdoc/demo/controller/TopicsController.java:23]: done

File: spring-boot-modules/spring-boot-swagger-2/src/main/java/com/baeldung/springdoc/demo/controller/TopicsController.java [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+		add(new Topic(1, "Topic1", "Topic1 Description"));
+		add(new Topic(2, "Topic2", "Topic2 Description"));
+		add(new Topic(3, "Topic3", "Topic3 Description"));
+	}};
+	
+	
+	@GetMapping(value = "/topics")
+	public ResponseEntity<List<Topic>> getAllTopics() {
+		ResponseEntity<List<Topic>> response = new ResponseEntity<>(topics, HttpStatus.OK);
+		return response;
```
[the remark this answers is not in the document]
Comment (JavierPeris) [reply spring-boot-modules/spring-boot-swagger-2/src/main/java/com/baeldung/springdoc/demo/controller/TopicsController.java]: I don't the changes here.

[the remark this answers is not in the document]
Comment (sdhiray7) [reply spring-boot-modules/spring-boot-swagger-2/src/main/java/com/baeldung/springdoc/demo/controller/TopicsController.java]: had missed it, done.

**Pull request merged**